server/routes/incidentDetails/incidentDetailsPage.ts

```diff
@@ -223,17 +223,16 @@ export default class IncidentDetailsPage {
 
     try {
       if (this.pageOptions.isEdit()) {
-        await this.saveToApiUpdate(postValues.draftId, incidentDetailsToSave, user as User)
+        const incidentRoleChanged =
+          postValues.originalIncidentRoleSelection !== incidentDetailsToSave.currentIncidentRoleSelection
+        const removeExistingOffences = incidentRoleChanged
```

Comment:
A bit weird, but I don't like implying a change in meaning as part of a function call